code/style.css

```diff
@@ -1,4 +1,193 @@
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: whitesmoke;
+  font-family: 'Arial Narrow' , sans-serif;
+}
+
+/*header*/
+header {
+  display: flex;
+  padding-bottom: 1%;
+}
+header img {
+  height: 8vw;
+  padding: 1% 2% 0 0;
+}
+.site-name {
+  align-self: center;
+  font-size: 3vw;
+}
+nav {
+  display: none;
+ }
+.hamburger {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 1% 1% 0 0;
+}
+.site-name:hover {
+color: #D3DEDC;
+}
+
+/*big news section*/
+.bignews-container {
+  display: box;
```

Comment:
I'm not sure but I don't think that you need to put display: box; when you have display: flex;